utils/httpUtils.ts

```diff
@@ -1,78 +1,93 @@
 import fetch from "node-fetch";
 
-export const formatUrlParams = (params: any) => {
-    const paramList = Object.entries(params)
-        .filter(([_, value]) => value !== undefined)
-        .map(([key, value]) => `${key}=${value}`);
-    if (paramList.length === 0) {
-        return "";
-    }
-    return "?&" + paramList.join("&");
-}
+/**
```

Comment:
I know this PR wasn't the original place where these resolvers were created, but any chance we could add some unit tests for the resolvers that it touches (similar to [the ZipLink test](https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/112930adf4dd3921427a794b33dfdc9536bc57a9/tests/zipLinkQuery.test.ts#L12))?  If there is a sample response for these resolvers, we might be able to just parse that and stub the http call response with it.